torch_xla/csrc/aten_autograd_ops.cpp

```diff
@@ -18,6 +18,51 @@ bool IsNonTrivialDilation(at::IntArrayRef dilation) {
 
 namespace aten_autograd_ops {
 
+torch::Tensor EinsumAutogradFunction::forward(
+    torch::autograd::AutogradContext* ctx, const c10::string_view equation,
+    at::TensorList tensors) {
+  ctx->saved_data["equation"] = equation;
```

Comment:
Let's just save the `std::string` instead of `string_view`. String view is a view on an existing string. I am not sure what does this means why we save a view while the real string object can be removed from stack once this call returns.

Reply:
Done